packages/superset-ui-legacy-preset-chart-big-number/package.json

```diff
@@ -42,5 +41,8 @@
     "@superset-ui/time-format": "^0.12.0",
     "@superset-ui/translation": "^0.12.0",
     "react": "^15 || ^16"
+  },
+  "devDependencies": {
+    "@types/shortid": "0.0.29"
```

Comment:
Let's put this in `dependencies`
https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies